src/index.js

```diff
@@ -124,7 +124,9 @@ export { MoreComponent           } from './atoms/MainRecipe/MoreLine';
 export { KKLogo                  } from './atoms/KKLogo';
 export { ButtonWithModal         } from './atoms/ButtonWithModal';
 export { NumberedListItem        } from './atoms/NumberedListItem';
-
+export { ActivityTime	         } from './atoms/SvgIcon/ActivityTime';
+export { Difficulty	      	     } from './atoms/SvgIcon/Difficulty';
+export { TotalTime	        	 } from './atoms/SvgIcon/TotalTime';
 
```

Comment:
Indentation

Reply:
Use spaces for alignment

Reply:
Strange. In IDE all ok

Reply:
This is alignment, and not indentation, so you should use spaces. Tabs vary in width depending on user settings.

Reply:
ok. got it

Reply:
about icons, we plan to use switch after that release

Reply:
That's OK, incremental changes are good.

I was wondering, thanks for mentioning it.